### PR TITLE
Fix audit db model and Application Subject

### PR DESCRIPTION
**Pull request by TiagoGraf89**

### CODE REVIEW

File: Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs [hunk 1]
```diff
@@ -54,6 +54,7 @@ public static IWebHostBuilder CreateWebHostBuilder<T>(string[] args)
             .ConfigureAppConfiguration((builderContext, config) =>
             {
                 config.AddEnvironmentVariables(prefix: EnvironmentPrefix);
+                config.AddJsonFile("appsettings.local.json", true, true); //load local settings
```
Comment (TiagoGraf89) [inline Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs]: This now enables us to have local configurations which are not in GitHub (store cert passwords or different settings)

Comment (sslaws) [reply Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs]: Wouldn't it be better to use secrets in our dev config?
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_169)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_169)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_169)

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_169)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_169)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_169)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_169)

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_169)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_169)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_169)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_169)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_169)

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_169)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_169)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_169)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_169)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_169)

File: Apps/Database/src/Migrations/AuditDb/AuditDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -45,7 +43,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<DateTime>("CreatedDateTime");
 
                     b.Property<string>("Trace")
-                        .HasMaxLength(20);
+                        .HasMaxLength(200);
```
Comment (FuriousLlama) [inline Apps/Database/src/Migrations/AuditDb/AuditDbContextModelSnapshot.cs]: +1 this one was breking some tests

File: Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs [hunk 3]
```diff
@@ -54,6 +54,7 @@ public static IWebHostBuilder CreateWebHostBuilder<T>(string[] args)
             .ConfigureAppConfiguration((builderContext, config) =>
             {
                 config.AddEnvironmentVariables(prefix: EnvironmentPrefix);
+                config.AddJsonFile("appsettings.local.json", true, true); // Loads local settings
```
Comment (FuriousLlama) [inline Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs]: We need to add that to gitignore as well if the purpose is to store local secrets

Comment (TiagoGraf89) [reply Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs]: Not going to implement secrets in this PR, and the local settings was already gitignored

File: Apps/Database/src/Migrations/AuditDb/AuditDbContextModelSnapshot.cs (hunk 2 above)
Comment (TiagoGraf89) [reply Apps/Database/src/Migrations/AuditDb/AuditDbContextModelSnapshot.cs]: Couldn't find any tests broken

**Pull request merged**